### PR TITLE
Fix ruff autofix parsing

### PR DESCRIPTION
**Pull request by tasgon**
Fixes a typo in the parser script and adds a workaround off-by-one error in ruff's output column locations in `fix` objects.

### CODE REVIEW

File: linters/ruff/ruff_to_sarif.py [hunk 1]
```diff
@@ -6,15 +6,15 @@
 results = []
 
 
-def get_region(entry):
```
Comment (TylerJang27) [inline linters/ruff/ruff_to_sarif.py:9]: The one piece I was missing--is this issue you were describing just specific to ruff/to this parser? If so, can you add a comment before line 59 and maybe before line 9 briefly describing the reason for this off-by-one fix?

**Review by TylerJang27 — COMMENTED** [review]
One comment, otherwise good

**Review by TylerJang27 — APPROVED** [review]
LGTM

File: linters/ruff/ruff_to_sarif.py (hunk 1 above)
Comment (tasgon) [reply linters/ruff/ruff_to_sarif.py:9]: This is something specific to Ruff. Just added comments

**Pull request merged**